File: oscourge_unity/Assets/Scripts/Networking/data/InputValues.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class InputValues {
    public InputValues (float h, bool j) {
        this.horizontal = h;
        this.jump = j;
    }

    public InputValues() {
        horizontal = 0;
        jump = false;
    }

    public float horizontal { get; set; }
    public bool jump { get; set; }
}

File: oscourge_server/src/ouscourge/net/data/InputValues.java
package ouscourge.net.data;

import org.json.JSONObject;

public class InputValues {
	private boolean jump;
	private float horizontal;
	
	public final static String JUMP = "jump";
	public final static String HORIZONTAL = "horizontal";
	
	public static InputValues valueOf(JSONObject obj) {
		InputValues newObj = new InputValues();
		newObj.setHorizontal(obj.getFloat(HORIZONTAL));
		newObj.setJump(obj.getBoolean(JUMP));
		return newObj;
	}
	
	public JSONObject toJson() {
		JSONObject json = new JSONObject();
		json.put(HORIZONTAL, this.horizontal);
		json.put(JUMP, this.jump);
		return json;
	}

	public float getHorizontal() {
		return horizontal;
	}

	public void setHorizontal(float horizontal) {
		this.horizontal = horizontal;
	}

	public boolean isJump() {
		return jump;
	}

	public void setJump(boolean jump) {
		this.jump = jump;
	}

}
